src/raymath.h

```diff
@@ -207,6 +207,13 @@ RMDEF float Vector2Length(Vector2 v)
     return result;
 }
 
+// Calculate vector square length
+RMDEF float Vector2SqrLength(Vector2 v)
```

Comment:
Please, could you review naming to `Vector2LengthSqr()` to better align with `Vector2Length()`?